src/components/button-group/ButtonGroup.scss

```diff
@@ -15,7 +15,7 @@
             margin-left: 0;
             padding-right: 11px;
             padding-left: 11px;
-            border-radius: 0 4px 4px 0;
+            border-radius: 0 $bdl-border-radius-med-size $bdl-border-radius-med-size 0;
```

Comment:
I'd suggest putting the modifier at the end to keep all these variables alphabetically together, which is similar to what we do for colors. Maybe like `bdl-border-radius-size-med`